src/meshdb/celery.py

```diff
@@ -13,37 +13,37 @@
 class LivenessProbe(bootsteps.StartStopStep):
     requires = {"celery.worker.components:Timer"}
 
-    def __init__(self, worker, **kwargs):
+    def __init__(self, worker, **kwargs):  # type: ignore[no-untyped-def]
```

Comment:
We can probably do types here, just use `dict` for the `kwargs` and `None` for the return type

Reply:
Didn't know that, thanks.

Reply:
Worker may be tricky but if it has a class you can just use the class and it’s essy